README.md

```diff
@@ -1,7 +1,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/Microsoft/AppCenter-SDK-Android/badge.svg?branch=develop)](https://coveralls.io/github/Microsoft/AppCenter-SDK-Android?branch=develop)
 [![GitHub Release](https://img.shields.io/github/release/microsoft/appcenter-sdk-android.svg)](https://github.com/microsoft/appcenter-sdk-android/releases/latest)
-[![Bintray](https://api.bintray.com/packages/vsappcenter/appcenter/appcenter/images/download.svg)](https://bintray.com/vsappcenter/appcenter)
 [![license](https://img.shields.io/badge/license-MIT%20License-00AAAA.svg)](https://github.com/microsoft/appcenter-sdk-android/blob/master/license.txt)
+[![Project Map](https://sourcespy.com/shield.svg)](https://sourcespy.com/github/microsoftappcentersdkandroid/)
```

Comment:
I'm sorry but we should not link 3rd party generated images - please add this image directly to our repo and provide instructions how to update it. Thank you.

Reply:
Thanks Lukas, valid point. I replaced the image with a standard one from shields.io (same used for license and others). No maintenance or updates needed for those. Hope this works now.

Reply:
Yes, thank you.